Big-Spender/readme.md

```diff
@@ -68,39 +75,59 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+INSERT SELECT ID, DESCRIPTION FROM SPENDS
+WHERE LOWER(DESCRIPTION) like '%fee%'
 ```
 
 **Farnoosh:** Hi, it's me again. It turns out we also need the transactions that have the expense area of 'Better Hospital Food'. Can you help us with that one?
 
 **You:** No worries. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+INSERT SELECT 
+    expense_area
+FROM 
+    expense_areas
+WHERE  lower(expense_area)
+like  '%food'
```

Comment:
It looks like you are only getting the expense_area of better hospital food, but we would need the transactions (ie- spends). So how would we go about doing this? 

hint: you will need to join another table ;)